xAPI.md

```diff
@@ -4080,7 +4080,7 @@ own internal storage, or need to persist state across devices.
 The semantics of the call are driven by the stateId parameter. If it is included, 
 the GET and DELETE methods will act upon a single defined state document 
 identified by "stateId". Otherwise, GET will return the available ids, and DELETE 
-will delete all state in the context given through the other parameters. This API has
+will delete all state in the context given through the other parameters. This Resource has
```

Comment:
I *think* this should be capitalized even if line 3566 is lowercase because it's talking about a specific resource?